javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java

```diff
@@ -65,7 +65,7 @@ private static CsmElement mix(CsmElement... elements) {
     }
 
     private static CsmElement memberAnnotations() {
-        return list(ObservableProperty.ANNOTATIONS, none(), none(), newline());
+        return list(ObservableProperty.ANNOTATIONS, newline(), none(), newline());
```

Comment:
Yes, it seems correct (I would love named parameters here...)

Reply:
We can convert everything to Kotlin.

Reply:
That would be brilliant!

Reply:
Just kidding :-D